src/auto-evo/AutoEvoExploringTool.tscn

```diff
@@ -762,80 +758,7 @@ margin_bottom = 486.0
 size_flags_horizontal = 3
 size_flags_vertical = 3
 
-[node name="SpeciesDetailPanel" type="PanelContainer" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer"]
-margin_left = 828.0
-margin_right = 1180.0
-margin_bottom = 508.0
-
-[node name="MarginContainer" type="MarginContainer" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel"]
-margin_left = 1.0
-margin_top = 1.0
-margin_right = 351.0
-margin_bottom = 507.0
-rect_min_size = Vector2( 350, 0 )
-custom_constants/margin_right = 10
-custom_constants/margin_top = 10
-custom_constants/margin_left = 10
-custom_constants/margin_bottom = 10
-
-[node name="VBoxContainer" type="VBoxContainer" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel/MarginContainer"]
-margin_left = 10.0
-margin_top = 10.0
-margin_right = 340.0
-margin_bottom = 496.0
-
-[node name="Label" type="Label" parent="GUI/VBoxContainer/HistoryReportSplit/Viewer/HBoxContainer/SpeciesDetailPanel/MarginContainer/VBoxContainer"]
```

Comment:
This label.

Reply:
I mentioned that label in the original post. I can leave it in if you would prefer, although I think it looks better in both places without it.

Reply:
I'm unsure about it, but I suppose it is fine then. However, I still want the SpeciesDetailsPanel to be a Control.